### DISCUSSION TITLE
So uh... how do i get a build??

### DISCUSSION THREAD

**Discussion by KingNoobInc** [RealMCoded/Rec.js#12] [Q&A] [answered]
i just got rec.js, and i dont have any builds.

**Comment by RealMCoded** [comment, marked as answer]
You have to source them yourself. I do not provide builds since they are copyright to Rec Room still.

**Reply by KingNoobInc** [reply]
shit

**Reply by CrypticBren** [reply]
Just use steamdb and depot downloader to get the build(s)

**Reply by fluffykanade** [reply]
oh, hi cryptic

**Reply by CrypticBren** [reply]
Hi chocomilk
